relay-single.ts

```diff
@@ -121,7 +125,7 @@ export class SingleRelayConnection implements Subscriber, SubscriptionCloser, Ev
     private constructor(
         readonly url: string,
         readonly wsCreator: (url: string, log: boolean) => BidirectionalNetwork | Error,
-        readonly signer: Signer | undefined,
+        readonly signer: InMemoryAccountContext | undefined,
```

Comment:
Obviously, you need
```
signer: Signer,
signer_v2?: Signer_V2 // optinoal
```
Do just shrink the parameter/input interface this easily. It will cause a whole class of incompability problems for no good.

We have discussed this topic several times.

If you haven't developed an instinct about this API design pattern yet.

Here is the Rule:
Making the input wider (a bigger set of types) is always compatible.
Making the output narrower (a smaller set of types) is always compatible.

The reverse is always incompatible. Meaning, will cause consumer of this API to fail compilation or runtime errors.